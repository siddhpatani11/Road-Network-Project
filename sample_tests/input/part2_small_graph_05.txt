15 105
0 2 19 B
0 3 17 G
0 3 48 B
0 5 43 B
0 8 20 R
0 9 6 G
0 9 21 R
0 10 26 R
0 11 12 R
0 13 15 B
1 2 25 R
1 2 43 G
1 4 21 R
1 6 42 R
1 6 46 R
1 8 5 G
1 8 31 R
1 10 11 B
1 11 50 G
1 13 13 B
1 13 30 G
2 0 17 B
2 1 44 G
2 4 41 B
2 5 7 R
2 5 5 G
2 6 25 B
2 9 51 B
2 10 1 B
3 4 39 B
3 5 21 B
3 9 44 R
3 10 39 R
3 14 25 R
4 0 19 B
4 1 27 G
4 2 27 G
4 7 10 B
4 9 33 B
4 10 15 G
4 12 4 B
5 2 44 G
5 4 2 R
5 9 26 G
5 11 29 G
5 14 17 B
6 1 31 B
6 2 23 R
6 4 3 R
6 8 24 G
6 11 25 G
7 4 51 G
7 4 11 B
7 5 33 G
7 10 39 G
7 13 31 B
8 1 33 R
8 7 34 B
8 11 36 B
8 12 15 R
8 12 47 R
8 14 9 G
9 1 18 B
9 6 26 G
9 6 19 R
9 8 28 G
9 10 29 G
9 12 12 G
9 12 13 R
9 14 33 R
10 6 12 G
10 6 28 B
10 14 4 G
10 14 21 R
11 0 41 R
11 2 46 B
11 4 31 R
11 5 39 R
11 10 46 B
12 1 31 B
12 3 32 G
12 4 7 B
12 5 2 R
12 10 43 B
12 11 36 G
12 11 10 G
13 0 29 G
13 1 31 B
13 2 11 R
13 3 29 B
13 5 45 G
13 5 35 R
13 6 19 B
13 7 28 B
13 7 27 B
13 9 47 G
13 9 48 G
13 12 38 R
13 12 32 G
14 1 32 G
14 4 22 B
14 5 12 G
14 5 19 B
14 8 31 R
14 12 44 G
