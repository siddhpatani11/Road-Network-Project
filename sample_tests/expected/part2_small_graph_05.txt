0 - 0 G 24 B 19 G 17 R 40 B 26 R 38 R 83 R 20 G 6 R 26 R 12 R 33 B 15 R 29 
